aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/AWSS3StoragePlugin.java

```diff
@@ -252,6 +265,7 @@ public void configure(@NonNull AmplifyOutputsData configuration, @NonNull Contex
             );
         }
 
+        this.configuredBuckets = storage.getBuckets();
```

Comment:
I would consider refactoring the private configure method that all other configure's go through just in case we introduce another path that supports multiple buckets.

```
private void configure(
    @NonNull Context context,
    @NonNull ResolvedStorageBucket defaultBucket,
    @NonNull Map<String, ResolvedStorageBucket> buckets
)
```

Can convert default region, bucket to a ResolvedStorageBucket.
Can convert buckets from config to a map of the friendly name to a ResolveStorageBucket.

Reply:
great input, I didn't think of it.  I've updated per suggestion.